spec/spec_helper_acceptance_local.rb

```diff
@@ -0,0 +1,59 @@
+# rubocop:disable Style/AccessorMethodName
+
+require 'serverspec'
+require 'puppet_litmus'
+require './spec/support/acceptance/helpers.rb'
+
+include PuppetLitmus
+
+RSpec.configure do |config|
+  include TargetHelpers
+
+  config.before(:suite) do
+    # Stop the puppet service on the master to avoid edge-case conflicting
+    # Puppet runs (one triggered by service vs one we trigger)
+    master.run_shell('puppet resource service puppet ensure=stopped')
+  end
+end
+
+# TODO: This will cause some problems if we run the tests
+# in parallel. For example, what happens if two targets
+# try to modify site.pp at the same time?
+def set_sitepp_content(manifest)
```

Comment:
Would be great to get this as a helper into Litmus after Enis' PR is merged. Just a thought.

Reply:
Will plan to update once we get this into Litmus.

Reply:
We don't need the TODO anymore btw, that was there back before the multi-node Litmus crap.